sdk/go/pulumi/config/require.go

```diff
@@ -29,29 +29,21 @@ func failf(format string, a ...interface{}) {
 	panic(fmt.Errorf(format, a...))
 }
 
-func require(ctx *pulumi.Context, key string, secret bool, use, insteadOf string) string {
+func require(ctx *pulumi.Context, key, use, insteadOf string) string {
```

Comment:
These changes seem unrelated?  Should they be  backed out of this PR?

Reply:
I have no idea how that got changed. I followed the contributing docs and that showed up changed after running the make commands. I included it since it said it was autogenerated. I can revert it to what's in the master branch